docs/how-to-run.adoc

```diff
@@ -2,24 +2,131 @@
 :toc: left
 :icons: font
 
-= Running Omicron (Non-Simulated)
+= Running Omicron
 
-Omicron is the control plane for an Oxide rack. It expects to execute
-on Helios systems, and Sleds use Helios-specific interfaces to manage
-resources.
+Omicron is the control plane for the Oxide system.  There are a few different ways to run Omicron depending on what resources you have available and how much of the stack you want to run using real components vs. simulated ones.  Generally speaking, it's easier to get things going and automate testing with simulated components, but of course much of the system's functionality is missing or faked up when using simulated components.
 
-If you're interested in running the control plane on other platforms, including
-Linux and Mac, refer to the guide on xref:how-to-run-simulated.adoc[running
-simulated Omicron].
+The most common configurations are:
 
-== Installing Prerequisite Software
+1. Using one or more simulated Sled Agents.  There are no real VMs.  All components listen on localhost and talk to each other directly.  The automated tests in this repo generally use this kind of deployment.  This is the only mode that's supported on non-illumos systems (i.e., Linux and MacOS).
```

Comment:
I realize this section is highlighting the most common configurations, but I think it's worth mentioning what the product version of Omicron looks like -- namely, how it differs from (2) in this list.